src/legacy/ui/public/courier/fetch/components/_shard_failure_modal.scss

```diff
@@ -0,0 +1,50 @@
+// set width and height to fixed values to prevent resizing when you switch tabs
+.shardFailureModal {
+     height: 75vh;
+     width: 768px;
+}
+
+// displaying a EuiCodeBlock inside a tab inside a modal, you've to set the width 
+// to prevent overlapping the modal when the content contains very long lines
+.shardFailureModal__codeblock {
+    width: 702px
+}
+
+// making EuiDescriptionList a bit more compact, there's not so much space inside a modal
+.euiDescriptionList.euiDescriptionList--column {
+    .shardFailureModal__descTitle {
+        width: 20%;
+        line-height: 1.1em;
```

Comment:
We add line-height is very specifically for readability. Please don't override this.

Reply:
thx, will never ever do it again :)